### ISSUE TITLE
Audit logs contain full user details, for organization audit logs this extends the user read permissions

### ISSUE THREAD

**Issue by Emyrk** [coder/coder#13949]
Audit logs currently expose a full user per log: https://github.com/coder/coder/blob/3685b189ac421b9bfd1894ff006ef4ba985a3dc3/codersdk/audit.go#L147-L147

This implicitly grants a "read" permission to `user`, even if the audit logs are scoped to an organization. Maybe we should strip down the user's returned fields.

**Comment by Emyrk** [comment]
Colin mentioned that we also log audit logs to stdout. Those can keep the stateful fields.

**Comment by bpmct** [comment]
I don't view it as a security concern that an org auditor can see org members' emails, username, login type, status, etc even if these properties are technically deployment-wide. Any tightening is nice-to-have IMO until we hear otherwise from our users.

**Comment by Emyrk** [comment]
@bpmct agreed. Some fields like `status` though are joined at the time of the call. So an audit log from last month is joined with the status from `today`. So the breaking change is the loss of more temporal fields

**Comment by Kira-Pilot** [comment]
@Emyrk can we take this out of the GA milestone based on Ben's feedback?

**Comment by Emyrk** [comment]
> @Emyrk can we take this out of the GA milestone based on Ben's feedback?

We can yes.
